### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by Muhammad-Juman-jarwar786** [community/community#66768] [GitHub Education]
### Hi everyone,

i had github student developer pack earlier and now i think it has been expired now the thing is i wanna renew the pack but the thing is it asked me to provide email issued by your school or university now the university email starts with usindh.edu.pk    but the university provided me the email which starts like students.usindh.edu.pk and i am using it and github is not recognizing my email so i am just stucked and do not know what to do so if anyone can tell me about this

**Comment by Shuhrat567** [comment]
battar bo'l

**Comment by Muhammad-Juman-jarwar786** [comment]
I was not able to understand your response would you mind clarifying it
please

**Comment by mantasu** [comment]
You may want to open a [ticket](https://support.github.com/request/education?tags=docs-generic) for GitHub Education
